app/models/CSVModels.js

```diff
@@ -0,0 +1,26 @@
+import csv from 'csvtojson';
+import mongoose from '../services/mongoose.js';
+
+const name = 'inspiration'
+
+// using the one csv in the repo for now.
+// later, let this be variable.
+const file_path = `./assets/decks/${name}/${name}.csv`
+
+export const rows = await csv().fromFile(file_path);
```

Comment:
Rather than export a promise, the best way to do this is to export a function that returns a promise, like so:

```
export async function rows() {
   return csv().fromFile(filePath);
}
```

By exporting a promise, you don't know whether the value of that export is a pending promise or a resolved promise. If you want to prevent multiple requests, you can locally cache the result:

```
let _rows;
export async function rows() {
  if (!_rows) {
    _rows = await csv().fromFile(filePath);
  }
  return _rows;
}
```